### ISSUE TITLE
`-enablesystemd` should find itself the installed script path

### ISSUE THREAD

**Issue by kumy** [bobafetthotmail/folder2ram#22]
I prefer to have manually installed scripts under `/usr/local/` folder. So I've put the script at `/usr/local/sbin/folder2ram`. Then when running the `systemd` configuration, it generates the services files using hard coded script path as `/sbin/folder2ram`.

It would have been nice if the script generate unit files using the real and current installation path.

It could have used `which` or `realpath $0`, no?

**Comment by bobafetthotmail** [comment]
Yes, it should not be a problem to make this change, but it's somewhat secondary so I can't guarantee I will do this in a short time.

If you want to do this and open a PR, I will merge it.
